src/main/resources/application.yaml

```diff
@@ -149,3 +146,45 @@ document_management.url: ${DM_URL:http://127.0.0.1:4506}
 idam.s2s-auth.totp_secret: ${IA_S2S_SECRET:AAAAAAAAAAAAAAAC}
 idam.s2s-auth.microservice: ${IA_S2S_MICROSERVICE:ia}
 idam.s2s-auth.url: ${S2S_URL:http://127.0.0.1:4502}
+
+### Email addresses
```

Comment:
I revert email address refactoring because the main Spring profile was changed to email-address from default which may have unknown implications